### ISSUE TITLE
__init__ command in help listing

### ISSUE THREAD

**Issue by honzajavorek** [apiaryio/black-belt#59]
Help listing looks like:

```
Usage: bb [OPTIONS] COMMAND [ARGS]...

  Black Belt: automate project The Apiary Way. Please provide a command.

Options:
  --help  Show this message and exit.

Commands:
  __init__
  gh        Handle github-related tasks and integrations.
  hip       Handle HipChat-related tasks and...
  init      Initialize application for usage.
  stage     Deploy current branch to staging
  t         Handle Trello-related actions and...
  version   Version
```

I suppose that `__init__` is there only accidentally.

**Comment by Almad** [comment]
@honzajavorek more like "I haven't figured out how to put it off". PRs welcome :)

**Comment by honzajavorek** [comment]
> PRs welcome :) 

Sure, that's the default :)

**Comment by honzajavorek** [comment]
@Almad imho it's this https://github.com/apiaryio/black-belt/blob/master/blackbelt/tasks.py#L12, you could avoid adding `__init__.py` file.

But I have no time to install dev bb now and make the PR. Either I'll get back during oncall or anyone else feel free to pick this up.

**Comment by Almad** [comment]
@honzajavorek Plausible, will try ;)

Well only think you need is virtualenv & `python setup.py develop`, but...:)

**Comment by honzajavorek** [comment]
@Almad yeah, on my new computer I'm definitely only JS dev so far. That's why I'm just filing issues. Once I have time to set everything up as I like it, I'll contribute again.

**Comment by honzajavorek** [comment]
Fixed by #69
